pages/api/sessions.ts

```diff
@@ -0,0 +1,39 @@
+import router from 'next/router'
+import { baseUrl } from 'pages/api/base'
+
+export const loginRequest = (formData: React.FormEvent<HTMLInputElement>) => {

```

Comment:
Please move the API folder outside the `pages` folder. Unless you are using `next/auth` which require it to be in the pages folder